hazelcast/src/main/java/com/hazelcast/cache/impl/AbstractCacheRecordStore.java

```diff
@@ -619,6 +619,10 @@ protected R updateRecord(Data key, R record, Object value, int completionId, Str
         }
     }
 
+    protected void updateRecordValue(R record, Object recordValue) {
```

Comment:
I have asked here because, I thought that if heap based cache (when hot-restart is enabled. btw I don't know if hot-restart is using also with heap based cache or not) relies on handling HB for updating record value by this method in `AbstractCacheRecordStore`, it may be problem. If you say that we handle it on enterprise already, I am OK.